runner/src/server/plugins/jwtAuth.ts

```diff
@@ -14,27 +14,57 @@ export function jwtAuthIsActivated(
   );
 }
 
+// keyFunc returns the key and any additonal context required to
+// passed to validate function (below) to validate signature
+// this is normally used to look up keys from list in a multi-tenant scenario
+const keyFunc = async function (decoded) {
+  const key = Buffer.from(config.rsa256PublicKeyBase64 ?? "", "base64");
+  console.log(
+    "Verifying token: '" +
+      JSON.stringify(decoded) +
+      "' with public key: '" +
+      config.rsa256PublicKeyBase64 +
+      "'"
+  );
+  return { key, additional: decoded };
+};
+
 // validate()
 // Checks validity of user credentials
 const validate = async function (decoded, request, h) {
   // This runs if the jwt signature is verified
   // It must return an object with an 'isValid' boolean property,
   // this allows the user to continue if true or raises a 401 if false
+  const credentials = decoded;
+  if (request.plugins["hapi-auth-jwt2"]) {
+    credentials.extraInfo = request.plugins["hapi-auth-jwt2"].extraInfo;
+  }
   if (!decoded.accountId) {
```

Comment:
are you just checking the account_id exists or is the plan to check its the correct account_id for an application?

Reply:
At the moment we're just checking the accountId exists. The next ticket I'm on now is checking that you can only get/put from the application store with the correct accountId jwt (rather than doing it here). The thinking being a user could only use the form runner if they already had the application form data from the application store, and the only danger it poses it that it puts back to the application store. So protecting it there is more straight forward

Reply:
yep makes sense 👍